packages/mui-base/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -607,6 +607,9 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
   const MarkLabel = components.MarkLabel || 'span';
   const markLabelProps = componentsProps.markLabel || {};
 
+  const Input = components.Input || 'input';
+  const inputProps = componentsProps.input || {};
```

Comment:
Note: this should change slightly after https://github.com/mui-org/material-ui/pull/30094 lands.

Reply:
Do you want me to rebase from this branch ?

Reply:
No, let's see which one will land first and we can rebase then from master :)